site/docs/v1/tech/apis/api-keys.adoc

```diff
@@ -0,0 +1,162 @@
+---
+layout: doc
+title: API Key APIs
+description: APIs for creating, retrieving, updating and deleting API Keys
+---
+
+== Overview
+
+[NOTE.since]
+====
+This API has been available since 1.26.0
+====
+
+The FusionAuth APIs are primarily secured using API keys. This page describes APIs that are used to manage API keys. Here are the APIs.
+
+* <<Retrieve an API Key>>
+* <<Create an API Key>>
+* <<Generate a New API Key From an Exising API Key>>
+* <<Update an API Key>>
+* <<Delete an API Key>>
+
+== Retrieve an API Key
+
+This API is used to retrieve a single API Key by unique Id.
+
+
+=== Request
+
+[.api-authentication]
+link:/docs/v1/tech/apis/authentication#api-key-authentication[icon:lock[type=fas]] Retrieve an API Key by Id
+[.endpoint]
+.URI
+--
+[method]#GET# [uri]#/api/api-key/`\{APIKeyId\}`#
+--
+
+==== Request Parameters
+
+[.api]
+[field]#APIKeyId# [type]#[UUID]# [required]#Required#::
+The unique Id of the API Key to retrieve.
+
+=== Response
+
+The response for this API contains a single API Key. The response is defined below along with an example JSON response.
+
+include::docs/v1/tech/apis/_standard-get-response-codes.adoc[]
```

Comment:
The 503 error is only returned when elastic search may be involved in the API response. I don't think that is the case here (though I may be wrong), so we want to bracket this with `never_search_error`, like so:

```
:never_search_error:
include::docs/v1/tech/apis/_standard-get-response-codes.adoc[]
:never_search_error!:
```

Please do this with all the other standard code includes.